src/Model/DriverStanding.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
File to remove ?

Reply:
sorry wrong drag n drop, i push a new version